product_customer_code_invoice/README.rst

```diff
@@ -0,0 +1,40 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============================
+Product Customer Code Invoice
+=============================
+Based on product_supplierinfo_for_customer, this module loads in every account invoice the
+customer code defined in the product.
+
+Bug Tracker
```

Comment:
Can you please add here the Usage section?

https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst